apps/web/modules/users/views/users-public-view.tsx

```diff
@@ -155,7 +155,7 @@ export function UserPage(props: InferGetServerSidePropsType<typeof getServerSide
                     <Link
                       prefetch={false}
                       href={{
-                        pathname: `/${user.username}/${type.slug}`,
+                        pathname: `/${user.profile.username}/${type.slug}`,
```

Comment:
Profile.username is supposed to have the correct username for an org member as well as non org member.